java/src/main/java/ai/rapids/cudf/HostColumnVector.java

```diff
@@ -379,6 +379,29 @@ public byte getByte(long index) {
     return offHeap.data.getByte(index * type.sizeInBytes);
   }
 
+  /**
+   * Get the byte list at index.
+   */
+  public byte[] getByteList(long index) {
+    assert type == DType.LIST;
+    assertsForGet(index);
+
+    int start = offHeap.offsets.getInt(index * DType.INT32.getSizeInBytes());
+    int end = offHeap.offsets.getInt((index + 1) * DType.INT32.getSizeInBytes());
+    int size = end-start;
+
+    // check if null or empty
+    if (start == end) {
+      if (isNull(index)) {
```

Comment:
In other places where we get a basic value we don't do null checks.  In fact `assertsForGet(index);` will do the null check for you if assertions are enabled and blow up if it is null.